resources/ext/ezbake.conf

```diff
@@ -21,7 +21,8 @@ ezbake: {
                          "mkdir -p /etc/puppet/manifests"],
                install: ["echo \\\"os-settings: {\\\"                         > $DESTDIR/$projconfdir/conf.d/os-settings.conf",
                          "echo \\\"    ruby-load-path: [$rubylibdir]\\\"      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf",
-                         "echo \\\"}\\\"                                      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf" ]
+                         "echo \\\"}\\\"                                      >> $DESTDIR/$projconfdir/conf.d/os-settings.conf" ],
+               pre-start-action: ["install --group={{user}} --owner={{user}} -d /var/run/puppet"]
```

Comment:
Ah shoot, I actually observed that behavior when the '-' was in front of the command, but removed it and somehow assumed the full path wasn't then necessary.